docs/references/architecture/adr-119-dog-mempool-gossip.md

```diff
@@ -0,0 +1,363 @@
+We have therefore opted to not reduce this value further. It does not impact the protocol as much as the target redundancy or the redundancy adjustment interval. 
+
+Part of the work on the protocol was extensive testing of its performance and impact on the network, as well 
+as the impact of the network configuration and load on the protocol itself. 
+Issue [\#4320] covers the experiments performed on DOG.
+
+We have also tested scenarios where we send each transaction to 100 of the 200 nodes. Typically we only send each transaction to one node. DOG was able 
+to reduce the number of duplicates in the system, even under a high load, even when the target redundancy is `1`. 
+
+After extensive testing, we did not find compelling evidence and differences in a variety of runs to warrant the
+tuning of the default values chosen. We have left the target redundancy and redundancy adjustment interval 
+as something operators can adjust so that users can experiment in their testnets. But none of our experiments
+found a degradation in performance and correctness for these value. 
+
+More details on the experiments used to make this decision
+ can be found in Issue [\#4320].
+
+
+
+## User recommendation
+
+- The Entire network should use DOG. Otherwise the impact will be minimal. 
+
+
+- The protocol implicitly favors faster routes, by cutting routes through peers that send the duplicate transaction at a later time.
+
+
+- If the frequency of `HaveTx` messages is too high, nodes will have too many routes cut.
+
+-  While we did not observe any single node missing out on transactions when the lowering the target 
+reduncancy event to `0.1`, we do not advise users to do this without a strong justification and testing. 
+
+- In networks that have slow connections between nodes (latencies bigger than 500ms), it is recommended to increase `config.dog_adjust_interval` to a higher value, at least as high as the maximum round-trip time (RTT) in the network.
+
+## Consequences
+
+Overall DOG reduces the bandwidth used by CometBFT, freeing up the network for consensus related messages. 
+For DOG to work as expected, the entire network should be using this, although it is not required for the network
+to behave correctly. It should however not be used in combination with the parameters `p2p.experimental_max_gossip_connections_to_non_persistent_peers` and `p2p.experimental_max_gossip_connections_to_persistent_peers` set to `true`.
+
+### Positive
+
+- Except for enabling DOG on all nodes, users do not have to change any other aspects of their application. 
+- Gossiping using DOG leads to a significant reduction in network traffic caused by transactions. 
+- Routes with lower latencies are implicitly favoured by cutting routes to peers that send the duplicate 
+- By cutting routes that go through a peer that sent the duplicates later in time, the protocol implicitly favors faster routes.
+
+### Negative
```

Comment:
We need to put something in Negative. I would add something like:
- It takes around 5 to 15 minutes for nodes to build a stable routing table, depending on network conditions and node configuration. This shouldn't pose a problem as it is expected for nodes to run for a long time.